frontend/src/app/orderTracking/page.tsx

```diff
@@ -7,30 +7,31 @@ import OrderReceiptManager, { Order } from "@/components/OrderReceiptManager";
 import { SelectionObject } from "@/components/SelectionObject";
 
 export default function OrderTrackingPage() {
-  const {currentSelection, isCurrentSelection, setCurrentSelection} =useSelection()
-  let fetchString = "api/orders?"
```

Comment:
Why was this removed? This was the fix for the last PR I pushed as not having this broke the build. Just revert this portion.